libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplatesParser.cs

```diff
@@ -27,14 +29,19 @@ public static class TemplatesParser
         /// <summary>
         /// option regex.
         /// </summary>
-        private static readonly Regex OptionRegex = new Regex(@"^> *!#(.*)$");
+        public static readonly Regex OptionRegex = new Regex(@"> *!#(.*)");
```

Comment:
>@"> *!#(.*)" [](start = 61, length = 12)

>/s* will capture any whitespace including tabs.

Reply:
yes

Reply:
I have changed the regex to '>\s*!#(.*)'.
There is no need to add \s before '>', because the lexer and parser help us make some limitations on it.